playground/main.js

```diff
@@ -126,4 +126,4 @@ if (typeof URL !== 'undefined') {
     render = false;
   }
 }
-render && renderPlaygroundWidget();
+render && ! window._isTestCafe && renderPlaygroundWidget();
```

Comment:
To prevent initial render with default options, remove and re-render with correct options

Reply:
It is already possible to skip initial render by passing `render: false`: https://github.com/okta/okta-signin-widget/blob/master/test/testcafe/spec/Interact_spec.js#L97

Reply:
Done